fastmri_recon/data/sequences/oasis_sequences.py

```diff
@@ -17,7 +17,8 @@ def _get_subject_from_filename(filename):
     subject_id = re.findall(r'OAS3\d{4}', base_name)[0]
     return subject_id
 
-class Oasis2DSequence(Sequence):
+
+class NIFTY2DSequence(Sequence):
```

Comment:
```suggestion
class NIFTI2DSequence(Sequence):
```

I think the file format is `nifti` not nifty